source/fundamentals/connection.txt

```diff
@@ -55,10 +55,26 @@ options. In the example, we set two connection options:
 ``maxPoolSize=20`` and ``w=majority``. To learn more about connection
 options, read the :ref:`golang-connection-options` section of this guide.
 
-The following code shows how you can use a connection string in a
-client to connect to MongoDB. In this example, the client uses 
-the :ref:`Stable API <golang-stable-api>` feature, though
-you can connect to MongoDB without this feature.
+Create a Client to Connect to MongoDB
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You need to create a client to connect to MongoDB, manage your
+connection, and run database commands. You can create a client by
+passing options to the ``Connect()`` method. These options
+include your connection string and any other client options.
+
+To learn more about creating a client in the {+driver-short+}, see the
+API documentation for `Client <{+api+}/mongo#Client>`__ and `Connect()
+<{+api+}/mongo#Connect>`__.
+
+In the options, you can set a Stable API server version to avoid
```

Comment:
Suggestion:

Avoid the options package ambiguity mentioned in the other comment.
```suggestion
You can set a Stable API server version option to avoid
```